Source/Core/Core/PowerPC/PPCAnalyst.cpp

```diff
@@ -106,9 +106,10 @@ bool AnalyzeFunction(u32 startAddr, Symbol& func, int max_size)
     }
     if (PPCTables::IsValidInstruction(instr))
     {
-      if (instr.hex == 0x4e800020)  // 4e800021 is blrl, not the end of a function
+      // 4e800021 is blrl, not the end of a function
+      if (instr.hex == 0x4e800020 || instr.hex == 0x4C000064)
       {
-        // BLR
+        // BLR or RFI
```

Comment:
The comment placement there looks a little odd to me regardless (both of them!) - mostly because the `if` above appears to check for the assembly, not the one inside; and the comment before the `if` describes something which isnt even part of the `if` check.